### PR TITLE
fix(files): show image if possible and css cleanup

### PR DESCRIPTION
**Pull request by josephmcg**
**What this PR does** 📖
- show image preview on file view if possible.
- clean up css

**Which issue(s) this PR fixes** 🔨
AP-902, AP-900

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Comment by phillsatellite** [comment]
Tested: @josephmcg should .heic images preview as well? 
Screen Shot 2022-02-25 at 12 17 48 PM

**Comment by josephmcg** [comment]
heic cannot be displayed in browser

File: components/views/files/view/View.vue [hunk 1]
```diff
@@ -33,10 +33,12 @@ export default Vue.extend({
   },
   computed: {
     ...mapState(['ui']),
-
     path(): string {
       return this.$Config.textile.browser + this.file.hash
     },
+    isImage() {
+      return this.file.name.match(this.$Config.regex.image)
```
Comment (molimauro) [inline components/views/files/view/View.vue:40]: Since is a boolean can we return `Boolean(this.file.name.match(this.$Config.regex.image))`? :)

**Comment by molimauro** [comment]
Only one super minor thing, other than that we're good!

File: components/views/files/view/View.vue (hunk 1 above)
Comment (josephmcg) [reply components/views/files/view/View.vue]: Thanks Mauro, good call. I added it to some other computed as well

**Comment by WanderingHogan** [comment]
@josephmcg I just tried uploading an heic to box and google drive and both preview it. you can create another ticket if Sara and Phil are okay with it to address it

**Review by WanderingHogan — APPROVED** [review]

**Comment by josephmcg** [comment]
We should probably handle HEIC in a separate ticket, I'm still not sure what the best course of action is

**Review by molimauro — APPROVED** [review]

**Review by phillsatellite — APPROVED** [review]

**Pull request merged**